src/components/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import { RefObject } from 'react';
+
+interface Props {
```

Comment:
It's better to name this type something like `HeaderProps`, so that the name describes this type the most precisely.